### ISSUE TITLE
import from

### ISSUE THREAD

**Issue by MountLion** [LingDong-/wenyan-lang#154]
```
習「foo」「bar」之術於「「somelib」」。名之曰「六脈神劍」。曰「淩波微步」。
```
==>
```
import foo as 六脈神劍, bar as 凌波微步 from "somelib"
```

```
邀「「洪七公」」。號之曰「北丐」。
使「北丐」施「降龍十八掌」之術於「西毒」。
```
==>
```
import "洪七公" as 北丐
北丐.降龍十八掌(西毒)
```

**Comment by cuixiping** [comment]
```
引「「算經」」之「正弦」「餘弦」。
「「算經」」有云：「正弦」「餘弦」。
```

**Comment by antfu** [comment]
Importing support is shipped in #290. Closing now. If you got any questions, free feel to reopen it.
